test/mongodb.test.js

```diff
@@ -82,10 +82,19 @@ describe('mongodb connector', function() {
       power: { type: String, index: true, unique: true },
       address: { type: String, required: false, index: { mongodb: { unique: false, sparse: true }}},
       description: { type: String, required: false },
-      geometry: { type: Object, required: false, index: { mongodb: { kind: '2dsphere' }}},
```

Comment:
@viki53 How about this one, can we preserve the index being defined in-line in the property definition?